helm/hpcc/templates/_helpers.tpl

```diff
@@ -761,6 +758,7 @@ Pass in dict with root, me
 {{- if has "dll" .me.access }}
 {{ include "hpcc.addDllVolume" . -}}
 {{- end }}
+{{ include "hpcc.addSecretVolumes" (dict "root" .root "categories" .secretsCategories) -}}
```

Comment:
trivial: perhaps addSecretVolumes should use "secretsCategories" too, then you could just pass in . instead of constructing dict